<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminUI</class>
 <widget class="QMainWindow" name="AdminUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#AdminUI{background-image: url(:/bg/p2)};

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="w" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1160">
       <property name="spacing">
        <number>7</number>
       </property>
       <property name="leftMargin">
        <number>11</number>
       </property>
       <property name="topMargin">
        <number>11</number>
       </property>
       <property name="rightMargin">
        <number>11</number>
       </property>
       <property name="bottomMargin">
        <number>11</number>
       </property>
       <item>
        <widget class="QWidget" name="nav" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget#nav{background-color: rgba(246,248,252,200);
border-radius: 20px;}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="133,40,40,40,40,133,40,40,40,40,133">
          <property name="spacing">
           <number>7</number>
          </property>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="InteractiveButtonBase" name="adminLogout">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>退出登录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="InteractiveButtonBase" name="readerInfo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>读者信息制定</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="InteractiveButtonBase" name="bookInfo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>书籍信息制定</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="InteractiveButtonBase" name="readerService">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>读者服务</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="InteractiveButtonBase" name="searchButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>搜索</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="InteractiveButtonBase" name="batchImportButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>批量导入</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="InteractiveButtonBase" name="submitChange">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>提交修改</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="InteractiveButtonBase" name="giveUpChange">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>放弃修改</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="styleSheet">
          <string notr="true">QStackedWidget#stackedWidget{background-color: rgba(240,240,240,220);border-radius: 16px;}</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="ripage">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_4" stretch="620,100">
             <item>
              <widget class="QTableView" name="readerInfoTable"/>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout" stretch="60,40">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="LabeledEdit" name="readerIdInput" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="LabeledEdit" name="readerNameInput" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="LabeledEdit" name="readerEmailInput" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="LabeledEdit" name="readerPhoneInput" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1">
                 <item>
                  <widget class="InteractiveButtonBase" name="deleteReader">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>120</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>删除读者</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="InteractiveButtonBase" name="editReader">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>120</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>修改读者</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="InteractiveButtonBase" name="addReader">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>120</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>新增读者</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="bipage">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout_9" stretch="600,100">
             <item>
              <widget class="QTableView" name="bookInfoTable"/>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_10" stretch="60,40">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="LabeledEdit" name="bookISBNInput" native="true">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="LabeledEdit" name="bookAuthorInput" native="true"/>
                 </item>
                 <item>
                  <widget class="LabeledEdit" name="bookKeywordInput" native="true"/>
                 </item>
                 <item>
                  <widget class="LabeledEdit" name="bookClassInput" native="true"/>
                 </item>
                 <item>
                  <widget class="LabeledEdit" name="bookNameInput" native="true"/>
                 </item>
                 <item>
                  <widget class="LabeledEdit" name="bookKindInput" native="true"/>
                 </item>
                 <item>
                  <widget class="LabeledEdit" name="bookPublishingInput" native="true"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,1,1">
                 <item>
                  <widget class="InteractiveButtonBase" name="deleteBook">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>120</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>删除书本</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="InteractiveButtonBase" name="editBook">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>120</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>修改书本</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="InteractiveButtonBase" name="addBook">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>120</width>
                     <height>40</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>新增书本</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="rspage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QWidget" name="gridWidget" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>720</width>
               <height>240</height>
              </size>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <property name="leftMargin">
               <number>7</number>
              </property>
              <property name="topMargin">
               <number>7</number>
              </property>
              <property name="rightMargin">
               <number>7</number>
              </property>
              <property name="bottomMargin">
               <number>7</number>
              </property>
              <property name="spacing">
               <number>7</number>
              </property>
              <item row="0" column="1">
               <widget class="InteractiveButtonBase" name="readerReturn">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>读者还书</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="InteractiveButtonBase" name="changePass">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>修改密码</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="InteractiveButtonBase" name="readerBorrow">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>读者借书</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="InteractiveButtonBase" name="lossOrbroke">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="text">
                 <string>丢失损坏处理</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="inputReaderIDpage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <property name="leftMargin">
            <number>11</number>
           </property>
           <property name="rightMargin">
            <number>11</number>
           </property>
           <item row="0" column="0">
            <widget class="QWidget" name="widget" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3" stretch="100,100,100">
              <property name="spacing">
               <number>7</number>
              </property>
              <property name="leftMargin">
               <number>7</number>
              </property>
              <property name="topMargin">
               <number>7</number>
              </property>
              <property name="rightMargin">
               <number>7</number>
              </property>
              <property name="bottomMargin">
               <number>7</number>
              </property>
              <item>
               <widget class="QLabel" name="label">
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="text">
                 <string>请输入读者ID</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="LabeledEdit" name="ridInput" native="true">
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>100</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="InteractiveButtonBase" name="readerIdConfirmButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>80</height>
                 </size>
                </property>
                <property name="text">
                 <string>确认</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InteractiveButtonBase</class>
   <extends>QPushButton</extends>
   <header location="global">interactive_buttons/interactivebuttonbase.h</header>
  </customwidget>
  <customwidget>
   <class>LabeledEdit</class>
   <extends>QWidget</extends>
   <header location="global">labeled_edit/labelededit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>readerInfo</tabstop>
  <tabstop>bookInfo</tabstop>
  <tabstop>readerService</tabstop>
  <tabstop>searchButton</tabstop>
  <tabstop>submitChange</tabstop>
  <tabstop>giveUpChange</tabstop>
  <tabstop>deleteReader</tabstop>
  <tabstop>editReader</tabstop>
  <tabstop>addReader</tabstop>
  <tabstop>adminLogout</tabstop>
  <tabstop>readerInfoTable</tabstop>
  <tabstop>deleteBook</tabstop>
  <tabstop>editBook</tabstop>
  <tabstop>addBook</tabstop>
  <tabstop>readerBorrow</tabstop>
  <tabstop>readerReturn</tabstop>
  <tabstop>lossOrbroke</tabstop>
  <tabstop>changePass</tabstop>
  <tabstop>readerIdConfirmButton</tabstop>
  <tabstop>bookInfoTable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
